rust_dev_preview/tls/src/main.rs

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+/*
+Purpose
+
+Shows how to use rustls, hyper-rustls, and webpki-roots to set up a connection using forcing a
```

Comment:
Same change as previous: "connection that forces a minimum..."

Reply:
fixed with a new sentence